contracts/service-registry/src/msg.rs

```diff
@@ -83,6 +83,11 @@ pub enum QueryMsg {
         service_name: String,
         verifier: String,
     },
+    #[returns(crate::contract::query::VerifierDetailsResponse)]
+    VerifierDetails {
+        service_name: String,
+        verifier: String,
+    },
```

Comment:
i wouldn't add a new query. Just change the return type of the `Verifier` query

Reply:
also, imo it's clearer to define the return type here in the msg.rs file. Makes things easier to read.